### DISCUSSION TITLE
XSD Validation - eForms Notice Editor Demo

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#340] [Q&A] [answered]
We have been doing different tests (for example with SDK 1.5 y 1.4, notice subtype 1) and we can't get it to perform the validation correctly.

`{
  "noticeUuid" : "744bfe92-7298-493e-8f64-3cdaa6a70683",
  "sdkVersion" : "1.4.1",
  "timestamp" : "2023-02-15T14:20:11.479751200Z",
  "Unsupported" : "This XSD validation feature is not supported in the editor demo for SDK version=1.4.1"
}`

The physical model is not generated correctly in the demo application?
Looking at the source code it seems that it is missing the mainXsdPathOpt element in the physical model.

Attached the visual and physical model

[visualModel.txt](https://github.com/OP-TED/eForms-SDK/files/10743939/visualModel.txt)
[physicalModel.txt](https://github.com/OP-TED/eForms-SDK/files/10743940/physicalModel.txt)

**Comment by rouschr** [comment, marked as answer]
Keep in mind this is the `develop` branch.
As it says:
`"This XSD validation feature is not supported in the editor demo for SDK version=1.4.1"`
Yes "mainXsdPathOpt" is missing due to this information only being available in SDK 1.6.0 (soon).
See my answer about the CVS validation in the other question as the problem is likely to be the XML tag order:
https://github.com/OP-TED/eForms-SDK/discussions/341
We did special developments not only to use it in the editor demo, but also in general to improve the SDK for other projects which want to avoid hardcoding the namespaces.

**Reply by AlexAS81** [reply]
Thanks, while we wait for version 1.6 of the sdk we will add the XSD path to be validated by hand.
